packages/ui/Scrollable/Scrollable.js

```diff
@@ -388,6 +388,10 @@ class ScrollableBase extends Component {
 			this.updateScrollbars();
 		}
 
+		if (isVerticalScrollbarVisible && !prevState.isVerticalScrollbarVisible) {
```

Comment:
In my opinion, at least we need to check if it is for Scroller or not because all items in a VirtualList could be re-rendered again by calling `forceUpdate`, then it make VirtualList rendering performance lower.

CC: @MikyungKim , @0x64

Reply:
Yeah I've been thinking about this as well. That's why this is just a placeholder PR. I'm still trying to figure out the best way to handle this. 

I'm not as familiar with `VirtualList` real world implementations, but VirtualList may not be hit as badly in normal scenarios. Most `VirtualList`'s would have a `ScrollBar` if it had enough items. So I assume that it would only be in a rare case.

Reply:
@rundmt 
When launching enact-internal-samples/qa-virtualgridlistnative, 53 items in the VirtualGridList are rendered. Then those items are rendered again by calling your `forceUpdate()` in your PR.

When rendering a VirtualList at the first time, we do not add a vertical or horizontal scroll bar. Then we render it again if we decide that we need any scroll bar because the item layout in VirtualGridList should be updated not to overlap the scrollbars and the items. At that time `componeneDidUpdate` function is called. If calling `forceUpdate()` in it, all items will be rendered again. All VirtualList and VirtualGridList works with this process when launching. So I would like to say that it is not in a rare case.

So that is the reason why I want to call `forceUpdate()` if it is a Scroller. By doing it, we could prevent increasing the VirtualList launch time.